com.github.virtual_puppet_project.vpuppr.yml

```
@@ -0,0 +1,47 @@
+id: com.github.virtual_puppet_project.vpuppr
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: OpenSeeFaceGD.x86_64
+finish-args:
+    - "--socket=x11"
+    - "--share=ipc"
+    - "--share=network" # to set up the deps for openseeface
+    - "--device=all"
+    - "--socket=pulseaudio"
+    - "--filesystem=home:ro"
+modules:
+
+  # --- Puppeteer ---
+  - name: puppeteer
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/bin/{resources,OpenSeeFaceFolder}
+      - install -Dm 644 -t /app/share/applications com.github.virtual_puppet_project.vpuppr.desktop
+      - install -Dm 644 com.github.virtual_puppet_project.vpuppr-128.png /app/share/icons/hicolor/128x128/apps/com.github.virtual_puppet_project.vpuppr.png
+      - mkdir -p /app/share/icons/128x128/apps/
```

```suggestion
      - install -Dm 644 com.github.virtual_puppet_project.vpuppr-128.png /app/share/icons/hicolor/128x128/apps/com.github.virtual_puppet_project.vpuppr.png
      - install -Dm 644 com.github.virtual_puppet_project.vpuppr-256.png /app/share/icons/hicolor/256x256/apps/com.github.virtual_puppet_project.vpuppr.png
```

1. install create the dir. 
2. you were trying to create it after.

as I said I'd keep the 256.